autogpt_platform/backend/backend/integrations/webhooks/stripe_test.py

```
@@ -0,0 +1,226 @@
+            resource="",
+            events=["customer.subscription.created"],
+            ingress_url="https://example.com/ingress",
+            secret="unused",
+        )
+
+
+@pytest.mark.asyncio
+async def test_deregister_webhook_deletes_endpoint() -> None:
+    response = AsyncMock()
+    response.status = 200
+    response.json = lambda: {}
```

Ruff (0.16.1)

[warning] Prefer `dict` over useless lambda

Replace with `lambda` with `dict`

(PIE807)